variables.tf

```diff
@@ -182,6 +182,22 @@ variable "cdn_host_redirects" {
   default     = []
 }
 
+variable "cdn_url_path_redirects" {
```

Comment:
Is it worth adding any validation to this or just leaving it for the user to implement correctly?

Reply:
As talked about since the validation comes with `azurerm` out of the box, I'll leave this one